interpreter/cling/lib/Interpreter/NullDerefProtectionTransformer.cpp

```diff
@@ -107,6 +107,11 @@ namespace cling {
     }
 
     NodeContext VisitStmt(Stmt* S) {
+      for (auto I = S->child_begin(), E = S->child_end(); I != E; ++I) {
+        if (*I)
+          NodeContext nc = Visit(*I);
+      }
+
       return NodeContext(S);
     }
 
```

Comment:
Do we really ever need to update the address of the elements inside a `CompoundStmt`? I don't think so... Then we don't need `VisitCompoundStmt()` at all...